code/modules/mob/living/carbon/human/human_verbs.dm

```diff
@@ -97,7 +97,7 @@
 		target.show_message("<span class='notice'>You hear a voice that seems to echo around the room: [say]</span>")
 	usr.show_message("<span class='notice'>You project your mind into [target.real_name]: [say]</span>")
 	log_say("[key_name(usr)] sent a telepathic message to [key_name(target)]: [say]")
-	for(var/mob/observer/ghost/G in world)
+	for(var/mob/observer/ghost/G in global.player_list)
```

Comment:
I could have sworn I did this already when I added `in world` to the CI checks.

Reply:
I did a dev merge so that's probably it? XD